platform/engineering/design-docs/2020-04-09-separate-content-build.md

```diff
@@ -168,8 +168,7 @@ The current front end build will be split up into two distinct builds:
     - Output: JavaScript and CSS bundles
     - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to a single S3 bucket with
-safeguards in place to ensure they don't override each other.
+The output of the content build will be deployed to one s3 bucket, and the output of the application build will be deployed to a different one. The separate bucket for the content build output will be new.
```

Comment:
It sounds like the application build will use the current S3 bucket used to hold both applications and content. Mind clarifying that?